WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -1215,15 +1215,28 @@ - (void)handleFollowConversationButtonTapped
     self.postHeaderView.isSubscribedToPost = newIsSubscribed;
 
     // Define success block
-    void (^successBlock)(void) = ^void() {
-        NSString *title = newIsSubscribed
-            ? NSLocalizedString(@"Successfully subscribed to the comments", @"The app successfully subscribed to the comments for the post")
-            : NSLocalizedString(@"Successfully unsubscribed from the comments", @"The app successfully unsubscribed from the comments for the post");
-
-        dispatch_async(dispatch_get_main_queue(), ^{
-            [generator notificationOccurred:UINotificationFeedbackTypeSuccess];
-            [weakSelf displayNoticeWithTitle:title message:nil];
-        });
+    void (^successBlock)(BOOL taskSucceeded) = ^void(BOOL taskSucceeded) {
+        if (taskSucceeded == NO) {
+            NSString *title = newIsSubscribed
+                ? NSLocalizedString(@"Unable to subscribe to the comments", @"The app failed to subscribe to the comments for the post")
```

Comment:
I wonder if these messages can be more in line with the button text, and other toast messages. Maybe like `Unable to follow conversation`, `Successfully followed conversation`, etc?